docs/docs/reference/dropped-features/package-objects.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: doc-page
+title: "Dropped: Package Objects"
+---
+
+Package objects
+```scala
+    package object p {
+      val a = ...
+      def b = ...
+    }
+```
+will be dropped. They are still available in Scala 3.0, but will be deprecated and removed afterwards.
+
+Package objects are no longer needed since all kinds of definitions can now be written at the top-level. E.g.
```

Comment:
One thing that package objects have over top-level definitions is that they can inherit traits and classes. I use this in e.g. Requests-Scala in order for the package-level API to expose a "default" configuration, while still having a trait that the user over the library can inherit from the create their own custom configurations of the API.